src/modules/BootGRUB2.rb

```diff
@@ -152,6 +152,9 @@ def Propose
         partition = prep_partitions.first
         if partition
           BootCommon.globals["boot_custom"] = partition
+          BootCommon.globals["boot_boot"] = "false"
+          BootCommon.globals["boot_root"] = "false"
+          BootCommon.globals["boot_mbr"] = "false"
```

Comment:
thats right. I am considering to change it, so stage1 propose boot location can propose custom on ppc. I already have similar bug report and it looks like better solution for me, as then you can replace proposal in BootGRUB2 with simple stage1 propose.